packages/angular_devkit/build_angular/src/tslint/index.ts

```diff
@@ -49,7 +53,7 @@ export default class TslintBuilder implements Builder<TslintBuilderOptions> {
 
     const version = tslint.Linter.VERSION && tslint.Linter.VERSION.split('.');
     if (!version || version.length < 2 || Number(version[0]) < 5 || Number(version[1]) < 5) {
-      throw new Error('TSLint must be version 5.5 or higher.');
+      throw new Error('TSLint must be version 5.13 or higher.');
```

Comment:
If we need a newer TSLint we need a migration for it. We shouldn't just show an error after update.

Even better than that would be to not need an update at all though... TSLint is going away in the future for ESLint, and on their side they have reverted the breaking change anyway. 

Maybe it's better to not use the optional argument at all and only require an update when TSLint becomes deprecated.

Reply:
Following our offline discussion, we don't to do this check nor a migration. 
More details in the PR description.

Reply:
This check isn't needed because the extra parameter is just ignored in versions 5.5 to 5.12.